packages/features/bookings/lib/bookingCreateBodySchema.ts

```diff
@@ -12,7 +12,7 @@ export const bookingCreateBodySchema = z.object({
   recurringEventId: z.string().optional(),
   rescheduledBy: z.string().email({ message: "Invalid email" }).optional(),
   start: z.string(),
-  timeZone: timeZoneSchema,
+  timeZone: z.string().refine((value: string) => isSupportedTimeZone(value), { message: "Invalid timezone" }),
```

Comment:
@emrysal was timeZoneSchema causing issues? I would like to keep a centralized schema for all timeZone validation.